File: April 2020/01 - Single Number/Solution.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        no_dupl = [];
        for i in nums:
            if(i not in no_dupl):
                no_dupl.append(i)
            else:
                no_dupl.remove(i)
        return no_dupl.pop()

File: April 2020/01 - Single Number/Solution.java
class Solution {
    public int singleNumber(int[] nums) {
      List<Integer> no_duplicate_list = new ArrayList<>();
  
      for (int i : nums) {
        if (!no_duplicate_list.contains(i)) {
          no_duplicate_list.add(i);
        } else {
          no_duplicate_list.remove(no_duplicate_list.size() - 1);
        }
      }
      return no_duplicate_list.get(0);
    }
  }